server/weaverbird/steps/fromdate.py

```diff
@@ -0,0 +1,20 @@
+from pandas import DataFrame
+from pydantic import Field
+
+from weaverbird.steps import BaseStep
+from weaverbird.types import ColumnName, DomainRetriever, PipelineExecutor
+
+
+class FromdateStep(BaseStep):
```

Comment:
just to be consistent with typescript step:
```suggestion
class FromDateStep(BaseStep):
```